test/server/admin/admin_test.cc

```diff
@@ -217,6 +222,242 @@ TEST_P(AdminInstanceTest, ConfigDumpMaintainsOrder) {
   }
 }
 
+// Test that using ?includeEds parameter adds EDS to the config dump.
```

Comment:
I'm thinking that based on the conventions already in https://www.envoyproxy.io/docs/envoy/latest/operations/admin, this should be renamed to `include_eds`.

Reply:
Modified.